src/Tools/BuildValidator/BuildConstructor.cs

```diff
@@ -27,17 +29,18 @@ internal class BuildConstructor
     {
         private readonly LocalReferenceResolver _referenceResolver;
         private readonly LocalSourceResolver _sourceResolver;
+        private readonly ILogger _logger;
 
-        public BuildConstructor(LocalReferenceResolver referenceResolver, LocalSourceResolver sourceResolver)
+        public BuildConstructor(LocalReferenceResolver referenceResolver, LocalSourceResolver sourceResolver, ILogger logger)
         {
             _referenceResolver = referenceResolver;
             _sourceResolver = sourceResolver;
+            _logger = logger;
         }
 
-        public Compilation CreateCompilation(MetadataReader metadataReader, string name)
+        public Task<Compilation> CreateCompilationAsync(CompilationOptionsReader optionsReader, string name)
```

Comment:
Method name says "Async" but the method isn't async.

Reply:
Fixed